java/spacewalk-java.spec

```diff
@@ -500,9 +500,9 @@ install -m 644 conf/rhn_java_sso.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config
 
 # Adjust product tree tag
 %if 0%{?sle_version} && !0%{?is_opensuse}
-sed -i -e 's/^java.product_tree.tag =.*$/java.product_tree.tag = Beta/' $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_java.conf
+sed -i -e 's/^java.product_tree_tag =.*$/java.product_tree_tag = Beta/' $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_java.conf
```

Comment:
Can you refresh my mind? What was the idea here, changing this manually when we are not doing SUSE Manager betas anymore?

Reply:
Yes. For SUSE Manager we remove the line completly. We have the final real value commited in the config (`SUMA4.1`). `Uyuni` stays.